### PR TITLE
remove installer update also from AutoYaST workflow (bsc#1024032)

### PR DESCRIPTION
**Pull request by jsrain**

### CODE REVIEW

File: control/control.CASP.xml [hunk 1]
```diff
@@ -441,13 +441,6 @@ configured accordingly to match the use case of the role. &lt;/p&gt;</label>
                 <enable_next>no</enable_next>
             </defaults>
             <modules config:type="list">
-                <!-- As soon as possible -->
-                <module>
-                    <label>Installer Update</label>
-                    <name>update_installer</name>
-                    <enable_back>no</enable_back>
```
Comment (lslezak) [inline control/control.CASP.xml:448]: I guess the disabled going back (`enable_back`) should be moved to the new first item.

**Review by lslezak — CHANGES_REQUESTED** [review]

File: control/control.CASP.xml (hunk 1 above)
Comment (jsrain) [reply control/control.CASP.xml:448]: Not necessarily, as neither autoinit nor autosetup steps are interactive

Comment (lslezak) [reply control/control.CASP.xml:448]: I see, then it's OK.

**Review by lslezak — APPROVED** [review]
LGTM

**Pull request merged**